### PR TITLE
[FR] Add `osu!_Community_Meetings`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `osu!_Community_Meetings`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/osu!_Community_Meetings/fr.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Il y a longtemps, dans le [passé lointain du jeu](https://twitter.com/ppy/status/1169256824052170755), l'équipe d'osu! avait l'habitude d'organiser régulièrement des réunions communautaires où tout le monde était invité à parler. Avec l'ampleur du jeu aujourd'hui, ce n'est plus possible, mais grâce à Discord, une sorte de système de file d'attente a été mis en place où les questions peuvent être abordées une par une. Si vous souhaitez poser des questions, assurez-vous d'avoir lu la FAQ ci-dessous, au cas où votre question aurait déjà été posée et aurait reçu une réponse !
+
+Actuellement, les réunions de la communauté osu! sont présidées par [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335) et [smoogipoo](https://osu.ppy.sh/users/1040328).
+
+Le prochain osu! community meeting est prévue pour le : **Le 28 novembre 2021 (dimanche 01:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:9]:
```suggestion
Le prochain osu! community meeting est prévu pour le : **Le 28 novembre 2021 (dimanche 01:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [compte Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# osu! community meetings
+
+Les **osu! community meetings** sont un groupe de discussion bimensuel organisé par l'[équipe d'osu!](/wiki/People/The_Team). L'objectif principal de ces réunions est de donner à chacun une chance de parler directement avec les développeurs et les personnes responsables de la gestion de la communauté pour soulever des questions à discuter ou à approfondir.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:3]:
```suggestion
Les **osu! community meetings** sont un groupe de discussion bimensuel organisé par l'[équipe d'osu!](/wiki/People/The_Team). L'objectif principal de ces meetings est de donner à chacun une chance de parler directement avec les développeurs et les personnes responsables de la gestion de la communauté pour soulever des questions à discuter ou à approfondir.
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Actuellement, les réunions de la communauté osu! sont présidées par [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335) et [smoogipoo](https://osu.ppy.sh/users/1040328).
+
+Le prochain osu! community meeting est prévue pour le : **Le 28 novembre 2021 (dimanche 01:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
+
+## S'impliquer
+
+Les osu! community meetings sont actuellement organisées sur une scène Discord sur le Discord [osu!dev](https://discord.gg/ppy), et diffusées en simultané sur [la chaîne Twitch de peppy](https://www.twitch.tv/ppy). Chaque réunion dure environ une heure et a lieu un dimanche sur deux à 01:00 UTC.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:13]:
```suggestion
Les osu! community meetings sont actuellement organisées sur une scène Discord sur le Discord [osu!dev](https://discord.gg/ppy), et diffusés en simultané sur [la chaîne Twitch de peppy](https://www.twitch.tv/ppy). Chaque meeting dure environ une heure et a lieu un dimanche sur deux à 01:00 UTC. 
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Planifiez vos points de discussion à l'avance et soyez aussi concis que possible.
+- Essayez de n'aborder que les questions qui concernent l'ensemble de la communauté, et pas seulement vous. 
+- La vérification sur osu!dev n'est pas nécessaire, mais peut augmenter vos chances d'être sélectionné pour des questions lors des sessions de questions-réponses.
+
+Nous avons l'intention de ne pas prendre plus de 5 minutes par question, donc comprenez que les questions auxquelles on a déjà répondu ou qui sont particulièrement génériques peuvent ne pas être choisies pour la discussion.
+
+## Archives
+
+Le premier osu! community meeting a eu lieu le 19 septembre 2021. Toutes les réunions ont été et continueront d'être enregistrées sur YouTube, accompagnées d'un document de synthèse.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:25]:
```suggestion
Le premier osu! community meeting a eu lieu le 19 septembre 2021. Tous les meetings ont été et continueront d'être enregistrées sur YouTube, accompagnées d'un document de synthèse.
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+| Réunion # | Lien | Résumé |
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:27]:
```suggestion
| Meeting n° | Lien | Résumé |
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+| :-: | :-- | :-- |
+| 0 | [19 septembre 2021](https://youtu.be/-LjNll8VYTM) | [Notes de réunion](https://gist.github.com/peppy/08b38cbac56a8b1e88133853a6a8d2fb) |
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:29]:
```suggestion
| 0 | [19 septembre 2021](https://youtu.be/-LjNll8VYTM) | [Document de synthèse](https://gist.github.com/peppy/08b38cbac56a8b1e88133853a6a8d2fb) |
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+| 1 | [3 octobre 2021](https://youtu.be/D3OZCm6szVE) | [Notes de réunion](https://docs.google.com/document/d/1-miPFZySYlYbtPW4G0828taycO1gh7dZ4icfSJCr4uA) |
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:30]:
```suggestion
| 1 | [3 octobre 2021](https://youtu.be/D3OZCm6szVE) | [Document de synthèse](https://docs.google.com/document/d/1-miPFZySYlYbtPW4G0828taycO1gh7dZ4icfSJCr4uA) |
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+| 2 | [17 octobre 2021](https://youtu.be/nLT-fLeogrw) | [Notes de réunion](https://docs.google.com/document/d/1wEQjzku_4IzvcWRanu0GkvWiPvb-IB7WzwU7Eyojj2E) |
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:31]:
```suggestion
| 2 | [17 octobre 2021](https://youtu.be/nLT-fLeogrw) | [Document de synthèse](https://docs.google.com/document/d/1wEQjzku_4IzvcWRanu0GkvWiPvb-IB7WzwU7Eyojj2E) |
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+| 3 | [31 octobre 2021](https://youtu.be/omvZbC7zJUI) | [Notes de réunion](https://docs.google.com/document/d/1bhaMx--chdys8AbzSDOItbv710SVhsww2BYPzQg2rvw) |
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:32]:
```suggestion
| 3 | [31 octobre 2021](https://youtu.be/omvZbC7zJUI) | [Document de synthèse](https://docs.google.com/document/d/1bhaMx--chdys8AbzSDOItbv710SVhsww2BYPzQg2rvw) |
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+| 4 | [14 novembre 2021](https://youtu.be/rEWgTY-L95Y) | [Notes de réunion](https://docs.google.com/document/d/1rhrz0zaxQfVK1UnvrcEqj4UMagTD0bPNh8wby8I38IA) |
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:33]:
```suggestion
| 4 | [14 novembre 2021](https://youtu.be/rEWgTY-L95Y) | [Document de synthèse](https://docs.google.com/document/d/1rhrz0zaxQfVK1UnvrcEqj4UMagTD0bPNh8wby8I38IA) |
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+Tout le monde est invité à participer ! Si vous avez une question ou quelque chose à ajouter à la discussion, vous pouvez lever la main sur la scène Discord pour rejoindre la file d'attente, ou écrire un message sur le fil Discord correspondant si vous n'êtes pas à l'aise pour parler, qui sera lu à haute voix et auquel on répondra. En outre, avant de poser une question, veuillez respecter les directives suivantes :
+
+- Planifiez vos points de discussion à l'avance et soyez aussi concis que possible.
+- Essayez de n'aborder que les questions qui concernent l'ensemble de la communauté, et pas seulement vous. 
+- La vérification sur osu!dev n'est pas nécessaire, mais peut augmenter vos chances d'être sélectionné pour des questions lors des sessions de questions-réponses.
+
+Nous avons l'intention de ne pas prendre plus de 5 minutes par question, donc comprenez que les questions auxquelles on a déjà répondu ou qui sont particulièrement génériques peuvent ne pas être choisies pour la discussion.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:21]:
```suggestion
Nous avons l'intention de ne pas prendre plus de 5 minutes par question, comprenez donc que les questions auxquelles on a déjà répondu ou qui sont particulièrement génériques peuvent ne pas être choisies pour la discussion.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Ganondorf-1 recheck please

**Review by TicClick — APPROVED** [review]
merging with the assumption that one review was enough

**Pull request merged**